rotkehlchen/tests/api/test_bitcoin.py

```diff
@@ -279,11 +282,27 @@ def test_add_delete_xpub_multiple_chains(rotkehlchen_api_server: 'APIServer') ->
         'xpub': xpub,
         'derivation_path': None,
     }
-    response = requests.put(api_url_for(
-        rotkehlchen_api_server,
-        'btcxpubresource',
-        blockchain='BCH',
-    ), json=json_data)
+
+    counter, account_num = 0, 23
+
+    def mock_check_haskoin(accounts: Sequence[BTCAddress]) -> dict[BTCAddress, tuple[bool, FVal]]:
+        """Temporarily mock the tx check since the haskoin api seems to be down as of 2025/06/16
+        TODO: Change the api we use for bch if haskoin remains down.
```

Comment:
Just change the api. It's completely down still. Check if it's dead. And if yes this is a bugfixes change, so it's not just a note.